Adds options to equiviliant exchange

mrjsavage opened the pull request:
You can now set the limit, the ordering, whether you want it to output as a csv and what team you want to limit your search to. The order of the options does not matter.
 
- Limit has to be a positive integer. A value of less than zero will result in no limit. Default is 10
- Team is as formated in the `bot.team_names`. Default is no team
- Output format is either csv or inline. Default is inline.
- Sort Direction is whether you want the bottom or the top of the data stack.

mrjsavage committed:
Adds options to equiviliant exchange

You can now set the limit, the ordering, whether you want it to output as a csv and what team you want to limit your search to.

tehstone commented on watcher/exts/playerstats.py:
```diff
@@ -186,13 +187,61 @@ async def _equivalent_exchange(self, ctx, *, info):
         info_split = info.split(",")
         raw_rating = info_split[0]
         player_name = info_split[1].strip()
+        print("hi")
```
hello! please remove

tehstone commented on watcher/exts/playerstats.py:
```diff
@@ -186,13 +187,61 @@ async def _equivalent_exchange(self, ctx, *, info):
         info_split = info.split(",")
         raw_rating = info_split[0]
         player_name = info_split[1].strip()
```
this is on me honestly, but we should add a check for array length before grabbing specific indexes from it. currently this throws errors straight to the log and doesn't tell the user what went wrong

tehstone commented on watcher/exts/playerstats.py:
```diff
@@ -218,22 +267,50 @@ async def _equivalent_exchange(self, ctx, *, info):
+              rating_star = player[5] * 5
+              rating = round((rating_star * 100)) / 100
+              response += f"**{o_player_name}**: {stars} ({team_name}) - {rating}⭐ {raw_rating}\n"
+          return await ctx.send(response)
```
discord message character limit is 2000 characters, if user indicates a large limit or less than 0 limit (unlimited) this response will fail with no indication to user what went wrong.

we *can* page the response (i think there's a helper method somewhere already to do this) but when used in busy public channels it gets very noisy hence the original limit of 10.

tehstone reviewed (changes requested):
couple of changes requested, thanks for the pr!

mrjsavage committed:
Output Tweaks

Captilised Players names and fixes a few things. And arranges the Top and Bottom correctly

mrjsavage committed:
Limits output to 10

Limits the output to 10 more than that.

mrjsavage committed:
Update playerstats.py

mrjsavage committed:
Adds check on less than 2 inputs

Checks if there are less than 2 inputs and returns an error if so.

mrjsavage committed:
Off by one error!

Opp threw an error when you had two commands.

mrjsavage committed:
Adds a help function to ee

`!ee help` gives a run down of the features.
